code/game/objects/items/weapons/implants/implant_traitor.dm

```diff
@@ -17,64 +17,69 @@
 	return dat
 
 /obj/item/implant/traitor/implant(mob/M, mob/user)
+
+	if(!user.mind.has_antag_datum(/datum/antagonist))  // if the user isnt an antagonist at all, don't let them use it
+		to_chat(user, "<span class='warning'><i>You lack the training required to use this type of implant.</i></span>")
+		return
```

Comment:
Return values are important for this proc.

For reference:
return 1 if the implant injects
return -1 if the implant fails to inject
return 0 if there is no room for implant

I appreciate that the default return value here is probably 0 and that gives us desired behavior, but it's best to explicitly return that value so future generations can understand what's going on here.

Reply:
I see, I'll add 0s to these